backend/src/api/blocks.ts

```diff
@@ -362,6 +362,14 @@ class Blocks {
           extras.expectedWeight = auditScore.expectedWeight;
         }
       }
+
+      extras.firstSeen = null;
+      if (config.CORE_RPC.DEBUG_LOG_PATH) {
+        const blockFirstSeen = await blocksRepository.$getFirstSeenTime(block.id);
```

Comment:
will a firstSeen time have been saved to the DB yet, when we make this lookup?

Reply:
Good catch, this is a remaining of me trying to debug while blocks were cached in redis 😅